src/components/stylingGuide.stories.mdx

```diff
@@ -0,0 +1,110 @@
+
+
+# Base Styles
+Every component extends either the [BaseSdkProps](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/61464e15175b0f88d141f968790524c5443ea5d1/src/components/sdk/Base/index.tsx) props or [BaseProps](https://github.com/aws/amazon-chime-sdk-component-library-react/blob/61464e15175b0f88d141f968790524c5443ea5d1/src/components/ui/Base/index.ts), which include the `css` and `classname` props. 
+
+# Overriding styles on a component
+There are multiple ways to override the base styling applied on SDK or UI components.
+
+Each UI or SDK component contains one or more dom elements, and in some cases the dom elements are nested within each other or dynamically rendered.
+In order select these dom elements in your stylesheets, you can refer to the individual component's documentation.
+
+## 1. Override styles using the `css` prop
+The first method is by providing the `css` prop to any of the components. This prop is meant to be used
+for minor changes and is not recommended if making more than a couple styling changes for a component.
+
+```
+// CSS prop 
+<Badge css="border: solid 1px black"/>
+```
+
+## 2. Override styles using the `className` prop
+
+The second method is to use the `classname` prop and apply your own classes to the top level dom element that, in some cases, contain child dom elements.
+This is the preferred method if you are making more than a couple styling changes to the component.
+
+The base styles were designed to hold a low specificity, however due to [this caveat](https://styled-components.com/docs/advanced#issues-with-specificity) with using styled-components, when you select dom elements in your own stylesheet, you'll need
+to use a higher specificity than just one class selector, otherwise the base styled applied by styled-components will override your styles.
+
+```
+import './style.scss'
+
+<Badge className="floating-badge"/>
+```
+Then, in your stylesheet, you need to specify the class name:
+
+```
+// ./style.scss
+.floating-badge.floating-badge {
+  // override the base styles in Badge component here
+}
+```
+Again, the reason for having to specify the class selector twice is due to [this caveat](https://styled-components.com/docs/advanced#issues-with-specificity) with styled-component. 
+
+## 3. Override styles using `styled-components`
+
+An alternative way you can override base styles is by wrapping an existing component from this library inside of another styled component.
+This is the preferred approach if you are familiar with styled-components.
+
+```
+import styled from 'styled-components';
+import { Label } from 'amazon-chime-sdk-component-library-react';
+
+// Create a new styled component based on one of the components from the component library
+let yourStyledLabel = styled(Label)`
+  border: solid 1px black;
+`
+```
+
+#  Unset styling on a component
```

Comment:
Should this section be above the first section in the guide? Ok to keep here just a discussion point.

Reply:
I think this is the smaller use case so I think it can go after - other component library documentations are the same